packages/components/src/components/icon/library.system.ts

```diff
@@ -17,6 +17,15 @@ export const icons = {
     <svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
       <path d="M12.743 6.333c-0.188-0.195-0.452-0.316-0.743-0.316s-0.555 0.121-0.743 0.315l-0 0-9 10c-0.137 0.17-0.22 0.388-0.22 0.626 0 0.552 0.448 1 1 1 0.276 0 0.526-0.112 0.707-0.293v0l8.257-9.173 8.257 9.173c0.181 0.181 0.431 0.292 0.707 0.292 0.552 0 1-0.448 1-1 0-0.238-0.083-0.456-0.222-0.628l0.001 0.002z"></path>
     </svg>
+  `,
+  close: `
```

Comment:
Did you clarify with @coraliefeil and @MarcMatthiae if this name will be the future one in the design system?